aws.go

```diff
@@ -185,11 +198,13 @@ func scrape(collector *cwCollector, ch chan<- prometheus.Metric) {
 func scrapeSingleDataPoint(collector *cwCollector, ch chan<- prometheus.Metric,params *cloudwatch.GetMetricStatisticsInput,metric *cwMetric,labels []string,svc *cloudwatch.CloudWatch) error {
 	resp, err := svc.GetMetricStatistics(params)
 	totalRequests.Inc()
-
+	
 	if err != nil {
 		collector.ErroneousRequests.Inc()
 		fmt.Println(err)
 		return err
+	} else {
+		collector.SuccessfulRequests.Inc()
 	}
```

Comment:
Else not required; as there is a return statement that returns on error.
```suggestion
	}
	
	collector.SuccessfulRequests.Inc()
```